packages/parrot-devtools/src/app/components/ScenarioSelector.jsx

```diff
@@ -48,6 +49,7 @@ const ScenarioSelector = ({
       <ul className="nav-menu">
         {scenarios.map(({ name }) => (
           <li className="nav-item" key={name}>
+            {/* eslint-disable-next-line jsx-a11y/anchor-is-valid, jsx-a11y/click-events-have-key-events, jsx-a11y/no-static-element-interactions */}
```

Comment:
Change this to button to so we don't have to write all these exceptions to eslint?